omod/src/main/webapp/pages/aijarLogin.gsp

```diff
@@ -141,6 +141,9 @@
         padding: 10px;
         background: beige;
     }
+    .login{
```

Comment:
@solemabrothers use the selection `#login-form input` rather than adding a class